docs/middleware/IndexRegistry.md

```diff
@@ -0,0 +1,39 @@
+# IndexRegistry
+
+This contract assigns each operator an index (0 indexed) within each of its quorums. If a quorum has $n$ operators, each operator will be assigned an index $0$ through $n-1$. This contract is used for AVSs that need a common ordering among all operators in a quorum that is accessible onchain. For example, this will be used in proofs of custody by EigenDA. This contract also keeps a list of all operators that have every joined the AVS for convenience purposes in offchain software that are out of scope for this document.
+
+## Flows
+
+### registerOperator
+
+The RegistryCoordinator for the AVS makes call to the IndexRegistry to register an operator for a certain set of quorums. The IndexRegistry will the assign the next index in each of the quorums the operator is registering for to the operator storing the following struct:
```

Comment:
typo: will the assign -> will assign

Reply:
79e1a902fe3d8cf4a53c6337d600a684a2ccf531